docs/core/project-sdk/msbuild-props.md

```diff
@@ -700,6 +700,9 @@ The `CodeAnalysisTreatWarningsAsErrors` property lets you configure whether code
 
 All code style rules that are [configured](../../fundamentals/code-analysis/overview.md#code-style-analysis) to be warnings or errors will execute on build and report violations.
 
+> [!NOTE]
+> If you install .NET 6 (or Visual Studio 2022, which includes .NET 6) but want to build your project using Visual Studio 2019, you might see new **CS8032** warnings if you have the `EnforceCodeStyleInBuild` property to `true`. To resolve the warnings, you can specify the version of the .NET SDK you want to build your project with (in this case, version 5) by adding a [global.json entry](../tools/global-json.md).
```

Comment:
Minor wording suggestion.  Would you really set version to 5 or would it be something like 5.0.304?
```suggestion
> If you install .NET 6 (or Visual Studio 2022, which includes .NET 6) but want to build your project using Visual Studio 2019, you might see new **CS8032** warnings if you have the `EnforceCodeStyleInBuild` property set to `true`. To resolve the warnings, you can specify the version of the .NET SDK to build your project with (in this case, version 5) by adding a [global.json entry](../tools/global-json.md).
```